clone_job.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
+
+import subprocess
+from subprocess import CalledProcessError
+import argparse
+import logging
+
+cmd = '/usr/share/openqa/script/clone_job.pl --skip-chained-deps --{0} {1} {2} {3}'
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--frm', default='openqa.suse.de')
+parser.add_argument('--params')
```

Comment:
As an Idea, we could use
```
parser.add_argument('params', nargs='*')
```
which is a positional argument and we could then make param like
```
param = ' '.join(args.params')
```
but a call would look like
```
clone_job.py --frm asfdasdf PARAM1=adfs PARAM2=gfawsda
```